data/Makefile

```diff
@@ -6,8 +6,11 @@ ensembl_biomart_geneids_grch37.p13.transcript_info.txt: ensembl_biomart_geneids_
 ensembl_biomart_transcripts.txt: ensembl_biomart_geneids_grch37.p13.transcript_info.txt
 	csvcut -tc transcript_stable_id,gene_stable_id,protein_stable_id,protein_length $< | csvsort -c transcript_stable_id,gene_stable_id,protein_stable_id,protein_length | uniq | csvformat -T > $@
 
-ensembl_biomart_transcripts.json: ensembl_biomart_transcripts.txt ensembl_biomart_pfam_grch37.p13.txt
-	cd ../scripts && python add_nested_domain_to_ensembl_transcript.py > ../data/$@
+ensembl_exon_info.txt: Homo_sapiens.GRCh37.gff3.gz
+	cd ../scripts && python transform_gff_to_tsv_for_exon_info_from_ensembl.py ../data/$< > ../data/$@
```

Comment:
assuming this last change ` ../data/$<`  was tested

Reply:
Yes this was tested